scripts/clean-build.sh

```diff
@@ -55,9 +55,9 @@ cleanup
 
 mkdir build || rm -rf build/*
 cd build
-cmake ..
-make -j10
-make test
+cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
We want to use the default here and not RelWithDebInfo. If we want RelWithDebInfo as the default it has to go into CMakeLists.txt

Reply:
You're right that RelWithDebInfo helps with debugging etc. However as clean-build.sh is used as CI entry point we should build always with the defaults here. Lets set RelWithDebInfo as default in CMakeLists.txt then.